LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2016 Bazaarvoice, Inc.
```

Comment:
2018

Reply:
nm

Reply:
https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated